frontend/src/styles/main.scss

```diff
@@ -1,4 +1,38 @@
-body, html {
-  background-color: deeppink;
-  height: 100%;
+$background-color: #171717;
+$card-color: #2F3138;
+$table-color: #212121;
+$border-color: #737784;
+$contrast-color: #FFFFFF;
+$alert-color: #DC3545;
+$success-color: #28A745;
```

Comment:
Is there a reason for using SCSS variables over CSS variables?

Reply:
> Is there a reason for using SCSS variables over CSS variables?

No, should we go with css variables?